hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/amrmproxy/AMRMProxyService.java

```diff
@@ -409,7 +420,7 @@ public void processApplicationStartRequest(StartContainerRequest request)
         return;
       }
       LOG.info("Callback received for initializing request "
```

Comment:
As we are touching, make it single line.

Reply:
I will fix it.